geth/node/node.go

```
@@ -229,3 +228,12 @@ func parseNodes(enodes []string) []*discover.Node {
 	}
 	return nodes
 }
+
+// parseNodesV5 creates list of discv5.Node out of enode strings.
+func parseNodesV5(enodes []string) []*discv5.Node {
+	nodes := make([]*discv5.Node, len(enodes))
+	for i, enode := range enodes {
+		nodes[i] = discv5.MustParseNode(enode)
```

do we really want to panic if any of the URLs is wrong?

---

yes, why would we want to continue?

---

that would silently crash the mobile app and it doesn't seem like a non-recoverable error.
I'd rather return an error here and then from `node.Start`, etc, so an API client can fix that.

---

it is the same behavior as with static nodes, frankly speaking i just don't want to mess with it in this pr